src/common/base/ICord.h

```diff
@@ -0,0 +1,240 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#pragma once
+
+#include <base/Base.h>
+
+namespace nebula {
+
+template <std::size_t kBlockSize = 1024>
```

Comment:
We'd better add a constraint to kBlockSize which should to be 2^n.  So that, we could replace mod with bit operation. You could take reference inside ConcurrentLRUCache.

Reply:
Good point.

Reply:
Had Done, please check.